### PR TITLE
TM-143 + TM-144 / download cv + professor see student's cv information

### PR DESCRIPTION
**Pull request by adelaidems**

### CODE REVIEW

File: code/server/routes/controller/student.js
Comment (fraR-coder) [inline code/server/routes/controller/student.js:1]: why did u change this? Isnt it better to use the id of the logged in user then passing it from the FE?

File: code/client/src/components/ApplicationDataCV.jsx [hunk 1]
```diff
@@ -36,6 +49,14 @@ function ApplicationData({ setShowData, handleErrors, setApplicationCV }) {
         }
     }, [selectedFile, examList]);//called every time the selected file is updated or the exam list is loaded
 
+    // useEffect(() => {
+    //   console.log("url", url);
+    //   const res = professorAPI.getCvFile(applicationInfo.cvId);
+    // }, [selectedFile, examList]);//called every time the selected file is updated or the exam list is loaded
+
+    const downloadCVFile = async () => {
```
Comment (fraR-coder) [inline code/client/src/components/ApplicationDataCV.jsx:52]: this is just for debugging right?

File: code/server/daoTeacher.js [hunk 2]
```diff
@@ -418,4 +418,46 @@ exports.getThesisAccepted = () => {
...
+                    );
+                } else {
+                    resolve(rows[0]);
+                }
+            }
+        );
+    });
+}
+
+exports.getCVDataByCVId = (cv_id) => {
```
Comment (fraR-coder) [inline code/server/daoTeacher.js]: this is never called?

**Review by fraR-coder — COMMENTED** [review]
all good just remove the console log and maybe add a check for the part about student id. Also add the auth.is LoggedIn to the download API

File: code/client/src/components/ApplicationDataCV.jsx (hunk 1 above)
Comment (adelaidems) [reply code/client/src/components/ApplicationDataCV.jsx:52]: yes yes, I forgot to delete it

**Pull request merged**